metrics-collector/README.md

```diff
@@ -0,0 +1,74 @@
+#Metrics-Collector
+
+Metrics-Collector is one of the components of CF `app-autoscaler`. It is used to collect application metrics from CF loggretator. The current version only supports memory metrics, it will be extended to include other metrics like throughput and response time at a later time. 
+
+## Getting started
+
+### System requirements:
+
+* Go 1.5 or above
+* Cloud Foundry relese 235 or later
+
+### Build and test
+
+1. Create a directory where you would like to store the source for Go projects and their binaries 
+2. Add this directory to your `$GOPAHT`
+3. create the directory `src/github.com/cloudfoundry-incubator` 
+3. change directory to `src/github.com/cloudfoundry-incubator`
+4. clone the `app-autoscaler` project: `clone git@github.com:cloudfoundry-incubator/app-autoscaler.git`
```

Comment:
git URLs only make sense for developers, we should put in https urls.
developers should have the following in their .gitconfig to do automatic rewrite
[url "git@github.com:"]
  pushInsteadOf = https://github.com/